degica/dockerfiles
# Add rails pack with Ruby 2.7.5

Simply a build pack using `ruby:2.7.5` with NodeJS installed for `bundle` and `assets` images of some of our recently updated repos (e.g HCS)

## How to Test
Watching CI pass. (check for this image:  https://github.com/degica/dockerfiles/runs/7224591667?check_suite_focus=true)
[HCS](https://github.com/degica/hcs/pull/497/commits/6a5a5d54c265bd6137050e505ebeb6ab6b69a585) will use this image to bundle gems and precompile assets. This image is essentially the same with what HCS built with successfully previously (both pulling from the official `ruby:2.7.5` image) but with added NodeJS and a bunch of other general dependencies same as our other ruby images (not necessary for HCS but to have a ready image for others to pull too).
